tests/urls.py

```diff
@@ -5,7 +5,7 @@
     urlpatterns = [path("admin", admin.site.urls)]
 except ImportError:
     # django < 2.0
-    from django.conf.urls import include, url
     from django.contrib import admin
+    from django.urls import include, path
```

Comment:
You can remove the whole try / except block here. And also handle these other entries: https://github.com/jazzband/django-fsm-log/search?q=except+ImportError